scripting/get5.sp

```diff
@@ -417,6 +426,8 @@ public void OnPluginStart() {
   g_OnSeriesInit = CreateGlobalForward("Get5_OnSeriesInit", ET_Ignore);
   g_OnSeriesResult =
       CreateGlobalForward("Get5_OnSeriesResult", ET_Ignore, Param_Cell, Param_Cell, Param_Cell);
+  g_OnSeriesResultWithoutWinners =
```

Comment:
I don't know if I NEED to create a forward for the new function (`Get5_OnSeriesResultWithoutWinners`) or if I can write it and use it without a forward. Maybe this is your point: show me that I can use the new function without a forward because it servers to other things. If you're saying that, I understand and I'm going to remove the forward.

But if you're saying that the `Get5_OnSeriesResultWithoutWinners` is too much and `Get5_OnSeriesResult` can handle this situation, let me try to explain in a more detailed way.

The `Get5_OnSeriesResult` function expects a `seriesWinner `, a `team1MapScore` and  a `team2MapScore`. Also, it retrieves the match stats with `Get5_GetMatchStats(kv);`, right? If so, it tests if there was a forfeit to keep this stats or throw them away. After that, the function creates the request to the API.

My point is that if there's no match (pretty much like a forfeit), there's no winners and no stats. So why check if was a forfeit if it wasn't? Same idea for match stats, why try to get the stats if there's no stats for that match?